README.md

```diff
@@ -42,6 +42,20 @@ Access the configuration values in the gem's code like this:
 Awesomeness.configuration.api_key # Whatever the user set
 ```
 
+To execute something the gem is configured:
```

Comment:
Could you change this to: "To execute something after the gem is configured:"

Reply:
Ah yes that's right!